combing/Spirale.scad

```diff
@@ -0,0 +1,32 @@
+$fa = 0.5;
```

Comment:
There's a typo in the file name here. It should be `spiral.scad`. Please use lowercase like the rest of the repository, too.